src/mumble/AudioOutputSpeech.cpp

```diff
@@ -357,23 +386,26 @@ bool AudioOutputSpeech::prepareSampleBuffer(unsigned int snum) {
 						}
 					}
 
-					update = (pow < (fPowerMin + 0.01f * (fPowerMax - fPowerMin)));
+					update = (pow < (fPowerMin + 0.01f * (fPowerMax - fPowerMin))); // Update jitter buffer when quiet.
 				}
+				// qlFrames.isEmpty() will always be true if using opus.
+				Q_ASSERT(qlFrames.isEmpty());
```

Comment:
Is this code part Opus exclusive?

Reply:
Ahhhh. Forgot to comment them out. I'm just trying to leave some useful marks to help people remove other deprecated codecs in the future.

Reply:
The `nullptr` comment is not the one that should contain that change though. Just check which commit introduced them and edit that one :)

Reply:
Done with some cherry-pick techniques. 

I have to say, formatting is no longer a fun thing for me anymore 😢 .

Reply:
> Done with some cherry-pick techniques.

It would probably have been easier if you simply rebased and then edited that specific commit :shrug: 

Anyways: Thanks for updating it :)